keycloak-plugin/sasl-plugin/pom.xml

```diff
@@ -8,6 +8,9 @@
   </parent>
   <modelVersion>4.0.0</modelVersion>
   <artifactId>sasl-plugin</artifactId>
+  <properties>
+    <maven.compiler.target>1.8.0</maven.compiler.target>
```

Comment:
For consistency with the other plugins,  shouldn't we demand source compatibility too?

Reply:
Hmm, looks like your comment refer to an out dated diff. If you refresh the view, what you request is there: https://github.com/EnMasseProject/enmasse/pull/2319/files#diff-32b0296924c065e6dd96b5268041380aR11